addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -27,6 +31,86 @@ class PaymentAcquirer(models.Model):
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
 
+    # === ACTION METHODS === #
+
+    def action_stripe_connect_account(self, menu_id=False):
+        """Create a Stripe Connect account and redirect the user to the next onboarding step.
```

Comment:
This is reeally nitpicky but, for the sake of consistency with how it's done pretty much everywhere in Odoo,
```suggestion
        """ Create a Stripe Connect account and redirect the user to the next onboarding step.
```